tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/AspNetMvcIntegration.cs

```diff
@@ -138,6 +141,18 @@ internal static Scope CreateScope(ControllerContextStruct controllerContext)
                             // extract propagated http headers
                             headers = httpContext.Request.Headers.Wrap();
                             extractedContext = tracer.TracerManager.SpanContextPropagator.Extract(headers.Value).MergeBaggageInto(Baggage.Current);
+
+                            InferredProxyScopePropagationContext proxyContext = null;
+                            if (tracer.Settings.InferredProxySpansEnabled)
+                            {
+                                proxyContext = InferredProxySpanHelper.ExtractAndCreateInferredProxyScope(tracer, headers.Value, extractedContext);
+                                if (proxyContext?.Scope is not null)
+                                {
+                                    SharedItems.PushScope(HttpContext.Current, HttpContextKey + ".proxy", proxyContext.Scope);
```

Comment:
The compiler may be taking care of this since `HttpContextKey` is a `const` and `".proxy"` is a string literal, but let's add a `string const` for it to make it nicer:

```diff
    internal static class AspNetMvcIntegration
    {
        internal const string HttpContextKey = "__Datadog.Trace.ClrProfiler.Integrations.AspNetMvcIntegration";
+       internal const string HttpContextKeyProxy = HttpContextKey + ".proxy";
```

Reply:
Put it into a `const`